src/app/screens/sendNft/index.tsx

```diff
@@ -18,93 +15,93 @@ import {
   validateStxAddress,
 } from '@secretkeylabs/xverse-core';
 import { useMutation } from '@tanstack/react-query';
-import { checkNftExists, isLedgerAccount } from '@utils/helper';
+import { StyledHeading, StyledP } from '@ui-library/common.styled';
+import { InputFeedback, InputFeedbackProps, isDangerFeedback } from '@ui-library/inputFeedback';
+import { checkNftExists } from '@utils/helper';
 import { getNftDataFromNftsCollectionData } from '@utils/nfts';
 import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useLocation, useNavigate, useParams } from 'react-router-dom';
 import styled from 'styled-components';
+import SendLayout from '../../layouts/sendLayout';
 
-const ScrollContainer = styled.div`
+const Container = styled.div`
   display: flex;
-  flex: 1;
   flex-direction: column;
-  overflow-y: auto;
-  &::-webkit-scrollbar {
-    display: none;
-  }
-  width: 360px;
-  margin: auto;
+  justify-content: space-between;
+  flex-grow: 1;
 `;
 
-const Container = styled.div({
-  display: 'flex',
-  flexDirection: 'column',
-  justifyContent: 'center',
-  alignItems: 'center',
-  flex: 1,
-});
-
-const NftContainer = styled.div((props) => ({
-  maxHeight: 148,
-  width: 148,
-  display: 'flex',
-  aspectRatio: 1,
-  justifyContent: 'center',
-  alignItems: 'center',
-  borderRadius: 8,
-  marginTop: props.theme.spacing(16),
-  marginBottom: props.theme.spacing(12),
-}));
+const StyledSendTo = styled(StyledHeading)`
+  margin-bottom: ${(props) => props.theme.space.l};
+`;
 
-const NftTitleText = styled.h1((props) => ({
-  ...props.theme.typography.headline_s,
-  color: props.theme.colors.white_0,
-  textAlign: 'center',
+const NextButtonContainer = styled.div((props) => ({
+  position: 'sticky',
+  bottom: 0,
+  paddingBottom: props.theme.spacing(12),
+  paddingTop: props.theme.spacing(12),
+  backgroundColor: props.theme.colors.elevation0,
 }));
 
-const BottomBarContainer = styled.div({
-  marginTop: 'auto',
-});
+const InputGroup = styled.div`
+  margin-top: ${(props) => props.theme.spacing(8)}px;
+`;
 
-const ButtonContainer = styled.div((props) => ({
+const Label = styled.label((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
   display: 'flex',
-  flexDirection: 'row',
-  marginLeft: '15%',
-  marginTop: props.theme.spacing(40),
+  flex: 1,
 }));
 
-const Button = styled.button((props) => ({
+const AmountInputContainer = styled.div<{ error: boolean }>((props) => ({
```

Comment:
we should consider making components from this to avoid code repetition, right?

Reply:
yeah true. could be the right time to make the input component in Ui library